packages/text-input/README.md

```diff
@@ -0,0 +1,58 @@
+# Text Input
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/text-input.svg)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/text-input
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/text-input
+```
+
+## Example
+
+```Javascript
+import TextInput from '@leafygreen-ui/text-input';
+
+const [value, setValue] = useState('');
+
+return (
+  <TextInput
+    label='Email'
+    description='Enter your email below'
+    placeholder='your.email@example.com'
+    onChange={event => setValue(event.target.value)}
```

Comment:
I believe we fixed this to be called with just the event sent back, if that's the case update docs accordingly :)

Reply:
Fixed